q2_vizard/assets/lineplot/index.html

```diff
@@ -67,7 +64,7 @@
       var y = data[view.signal('yField')];
 
       // add attrs to each mark
-      mark.setAttribute('class', 'mark-' + id);
+      mark.setAttribute('class', 'data-' + id);
```

Comment:
```suggestion
      mark.setAttribute('data-id', id);
```
Sorry, I had meant this to make it consistent.

Reply:
ahh, gotcha - yep, i can change that!